### PR TITLE
bump jackson version

### PR DESCRIPTION
**Pull request by 2bitoperations**

### CODE REVIEW

File: rdbi-parent/pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jackson.version>2.9.8</jackson.version>
+        <jackson.version>[2.9.10.1,)</jackson.version>
```
Comment (phutwo) [inline rdbi-parent/pom.xml]: why does rdbi need jackson?

Comment (phutwo) [reply rdbi-parent/pom.xml]: objects should be opaque no?

Comment (2bitoperations) [reply rdbi-parent/pom.xml]: there is a cache impl that "does it for you".
long term we can tear it out, but I just wanted to resolve the security alert for today.

Comment (phutwo) [reply rdbi-parent/pom.xml]: is jackson set up as bring your own?

Comment (2bitoperations) [reply rdbi-parent/pom.xml]: i'm not sure of a way to express that in _this_ maven pom, but in practice our usages of RDBI exclude jackson as a transient dep and specify the version that "we" want.

**Pull request merged**